### DISCUSSION TITLE
An example of Hydrogen atom 3D orbital

### DISCUSSION THREAD

**Discussion by 1sHydrogenAtom** [pyvista/pyvista#4799] [Q&A] [answered]
I want to plot a 3D orbital of a hydrogen atom. I copied the exact code written in the documentation and ran the code. But it shows 'examples.load_hydrogen_orbital has no attribute' and I cannot plot the example. I tried to find a file named 'examples.load_hydrogen_orbital'. But I could not find it.

Do you know why? I already installed sympy as well.

```python
import numpy as np

import pyvista as pv
from pyvista import examples

grid = examples.load_hydrogen_orbital(3, 2, -2)
grid

pl = pv.Plotter()
vol = pl.add_volume(grid, cmap='magma', opacity=[1, 0, 1])
vol.prop.interpolation_type = 'linear'
pl.camera.zoom(2)
pl.show_axes()
pl.show()

```

**Comment by tkoyama010** [comment]
Thanks for the report. I have confirmed that the code works in my environment.
To compare your environment with mine, please run the following code wherever you are experiencing it, and paste the output to this discussion.
```python
# Get system info
import pyvista as pv
print(pv.Report())
```
My environment is:
```bash
--------------------------------------------------------------------------------
  Date: Thu Aug 24 05:17:01 2023 JST

                OS : Linux
            CPU(s) : 20
           Machine : x86_64
      Architecture : 64bit
       Environment : Python
        GPU Vendor : Intel
      GPU Renderer : Mesa Intel(R) Graphics (ADL GT2)
       GPU Version : 4.6 (Core Profile) Mesa 23.0.4-0ubuntu1~23.04.1
  MathText Support : False

  Python 3.11.4 (main, Jun  9 2023, 07:59:55) [GCC 12.3.0]

           pyvista : 0.41.1
               vtk : 9.2.6
             numpy : 1.25.2
        matplotlib : 3.7.2
            scooby : 0.7.2
             pooch : v1.7.0
--------------------------------------------------------------------------------
```

**Reply by 1sHydrogenAtom** [reply]
Hi, thank you for answering me. This is the output.

--------------------------------------------------------------------------------
 Date: Thu Aug 24 13:41:19 2023 W. Europe Daylight Time

                OS : Windows
            CPU(s) : 12
           Machine : AMD64
      Architecture : 64bit
               RAM : 31.8 GiB
       Environment : IPython
       File system : unknown
        GPU Vendor : Intel
      GPU Renderer : Intel(R) UHD Graphics 630
       GPU Version : 4.5.0 - Build 27.20.100.9664

  Python 3.9.7 (default, Sep 16 2021, 16:59:28) [MSC v.1916 64 bit (AMD64)]

           pyvista : 0.33.3
               vtk : 9.1.0
             numpy : 1.20.3
           imageio : 2.9.0
           appdirs : 1.4.4
            scooby : 0.5.11
        matplotlib : 3.4.3
         pyvistaqt : 0.11.0
             PyQt5 : 5.9.2
           IPython : 7.29.0
             scipy : 1.7.1
              tqdm : 4.62.3

--------------------------------------------------------------------------------

**Reply by tkoyama010** [reply, marked as answer]
Thanks for the confirmation. `load_hydrogen_orbital` was added in v0.38.1 and does not work in v0.33.3. Could you upgrade the PyVista version with the following command and run your script to check again?
```windows
pip install --upgrade pyvista
```

**Comment by 1sHydrogenAtom** [comment]
Thank you! It works!!
